test/controllers/drivers_controller_test.rb

```diff
@@ -5,68 +5,114 @@
 
   describe "index" do
     it "responds with success when there are many drivers saved" do
-      # Arrange
-      # Ensure that there is at least one Driver saved
-
+      Driver.create(name: "CheezitMan", vin: "ABCDEFGHIJKLMNOPQ")
       # Act
-
+      get drivers_path
       # Assert
-
+      must_respond_with :success
     end
 
     it "responds with success when there are no drivers saved" do
       # Arrange
       # Ensure that there are zero drivers saved
 
       # Act
-
+      get drivers_path
       # Assert
-
+      must_respond_with :success
     end
   end
 
   describe "show" do
     it "responds with success when showing an existing valid driver" do
+      driver = Driver.create(name: "CheezitMan", vin: "ABCDEFGHIJKLMNOPQ")
       # Arrange
-      # Ensure that there is a driver saved
 
+      # Ensure that there is a driver saved
       # Act
-
+      get driver_path(driver.id)
       # Assert
-
+      must_respond_with :success
     end
 
     it "responds with 404 with an invalid driver id" do
       # Arrange
       # Ensure that there is an id that points to no driver
 
       # Act
-
+      get driver_path(Driver.last.id + 1)
```

Comment:
Just a note that you can't guarantee that the last driver's id is the max driver id.  Postgres will maintain an order, but not necessarily numeric.  If you want to do this it's best to do it this way.  You also need to create some drivers for this to work. 

```suggestion
      get driver_path(Driver.all.order(:id).last.id + 1)
```

also you were missing .all.  That's why the test failed.